reduct/boolean-reduct/rte-helpers.metta

```diff
@@ -36,19 +36,26 @@
 )
 
 ;a function to check whether an n-ary expression is consistent or not.
+;If any negation pairs are found (via areNegations), the function returns False, indicating inconsistency.
+
+(= (areNegations ($a $b))
+    (or (unify $a (NOT $b) True False)
```

Comment:
I used unify instead of if-else because it simplifies the logic by directly matching patterns and returning a boolean result. But the other approach works just as well, should I edit it?

Reply:
Yeah the simpler the logic the better. Using `unify` here implies that the input isn't single but a non deterministic list.

Reply:
It has been modified

Reply:
I don't see the changes here though, did you push it?

Reply:
Sorry about that, it wasn't synced. Its been updated now